### PR TITLE
Adding version 2018-07-01 for Microsoft.Media

### PR DESCRIPTION
**Pull request by quintinb**
I had a merge issue with the branch for my previous pull request (https://github.com/Azure/azure-rest-api-specs/pull/3831).  Thus, I am copying my changes from that branch into this fork and submitting a new pull request to just pick up my changes and not the result of the bad merge.  My previous pull request was already reviewed before the bad merge and is ready to be merged into master.

Please let me know if you have any questions or concerns.

### CODE REVIEW

**Review by anuchandy — APPROVED** [review]

**Comment by anuchandy** [comment]
Both SDK and ARM approval is already received for same changes in the [PR](https://github.com/Azure/azure-rest-api-specs/pull/3831)

File: specification/mediaservices/resource-manager/readme.md [hunk 1]
```diff
@@ -178,6 +195,14 @@ Please also specify the `--go-sdk-folder=<path to the root directory of your azu
 output-folder: $(go-sdk-folder)/services/preview/mediaservices/mgmt/2018-06-01-preview/media
 ```
 
+### Tag: package-2018-07 and go
+
+These settings apply only when `--tag=package-2018-07 --go` is specified on the command line.
+Please also specify the `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2018-07' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/mediaservices/mgmt/2018-07-01/media
```
Comment (jhendrixMSFT) [inline specification/mediaservices/resource-manager/readme.md]: Did you intend to release this as a preview package?  All of the swaggers are in the stable directory.

Comment (quintinb) [reply specification/mediaservices/resource-manager/readme.md]: No, this should be a stable package.  If there is anything I need to do in the md files please let me know.

Comment (jhendrixMSFT) [reply specification/mediaservices/resource-manager/readme.md]: For Go simply remove the `preview/` part from the `output-folder`.  Not sure about the other languages.

Comment (quintinb) [reply specification/mediaservices/resource-manager/readme.md]: Fixed.

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Pull request merged**